sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/entries/CreateNewTravelEntrySteps.java

```diff
@@ -226,6 +259,58 @@ public CreateNewTravelEntrySteps(
               "User name is invalid");
           softly.assertAll();
         });
+
+    When(
+        "^I check if pick or create person popup in travel entry is displayed$",
+        () -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(
+              PICK_OR_CREATE_PERSON_TITLE_DE); // wait for popup
+          String expectedTitle = "Person ausw\u00E4hlen oder erstellen";
+          String checkPopupTitle =
+              webDriverHelpers
+                  .getTextFromWebElement(PICK_OR_CREATE_PERSON_TITLE_DE)
+                  .toLowerCase(Locale.GERMAN);
+          softly.assertEquals(
+              checkPopupTitle,
+              expectedTitle.toLowerCase(Locale.GERMAN),
+              "Wrong popup title for Pick or create person");
+          softly.assertAll();
+        });
+
+    When(
+        "^I check Pick an existing case in Pick or create person popup in travel entry$",
+        () -> webDriverHelpers.clickOnWebElementBySelector(PICK_A_EXISTING_PERSON_LABEL_DE));
+
+    When(
+        "^I click confirm button in popup from travel entry$",
+        () -> webDriverHelpers.clickOnWebElementBySelector(SAVE_POPUP_CONTENT));
+
+    When(
+        "I choose an existing case while creating case from travel entry",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(
+              PICK_A_EXISTING_CASE_LABEL_DE); // wait for popup
+          String expectedTitle = "Fall ausw\u00E4hlen oder erstellen";
+          String checkPopupTitle =
+              webDriverHelpers
+                  .getTextFromWebElement(PICK_OR_CREATE_PERSON_TITLE_DE)
+                  .toLowerCase(Locale.GERMAN);
+          softly.assertEquals(
+              checkPopupTitle,
+              expectedTitle.toLowerCase(Locale.GERMAN),
+              "Wrong popup title for Pick or create a case");
+          softly.assertAll();
+          webDriverHelpers.clickOnWebElementBySelector(PICK_A_EXISTING_CASE_LABEL_DE);
+        });
+
+    When(
+        "^I check if created travel entries are listed in the epidemiological data tab$",
+        () -> {
+          webDriverHelpers.isElementDisplayedIn20SecondsOrThrowException(
+              By.xpath("//div[text()='" + entryPoint + "']"));
```

Comment:
we should take this value: entryPoint  from a POJO, so we'll make the code reusable